app/views/health/patients/_filter.haml

```diff
@@ -40,7 +40,7 @@
           Engagement Period Ending
 
       - unassigned_selector = OpenStruct.new(name: 'Only Unassigned', id: 'unassigned')
-      - selection_candidates = [unassigned_selector, current_user] + current_user.team_mates
+      - selection_candidates = [unassigned_selector, current_user] + User.care_coordinators.to_a
```

Comment:
This means anyone can pick any care coordinator right?

Reply:
Yes, but since it only filters the list of clients they can see, I didn't see any concern in that.